project-words.txt

```diff
@@ -667,4 +670,4 @@ Postbot
 Postbots
 permissionlessly
 redcircle
-bluecircle
\ No newline at end of file
+bluecircle
```

Comment:
Didn't seem like this one got put in the right place? So the conflict is still there :(